src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -242,6 +251,29 @@ services.AddHttpClient(
 Note: The single instance returned by `HttpClientFactory` is reused by all
 export requests.
 
+## Experimental features
+
+The OTLP exporter contains the following experimental features that are
```

Comment:
anything user need to do to enable experimental feature other than using the pre-release package?

Reply:
No